lib/jobs/parsedProfileProcessor.js

```diff
@@ -0,0 +1,100 @@
+require('dotenv').config();
+const mongoose = require('mongoose');
+const cheerio = require('cheerio');
+const ProfileReport = require('../models/ProfileReport');
+
+//TODO
+// fix issue with annual training summary
+
+module.exports = async(job) => {
+  mongoose.connect(process.env.MONGODB_URI, {
```

Comment:
instead of passing the entire `RawProfileReport` to this job I would pass just the `RawProfileReport`s id. then:

```js
try {
  const { html } = RawProfileReport.findById(job.data.id);
  const $ = cheerio.load(html);
  return ProfileReport.create(parseProfile($);
} finally {
  await mongoose.connection.clone()
}
```